packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx

```diff
@@ -61,22 +62,24 @@ export class SelectedPeopleList extends BasePeopleSelectedItemsList {
   protected renderItems = (): JSX.Element[] => {
     const { items } = this.state;
     // tslint:disable-next-line:no-any
-    return items.map((item: any, index: number) => this._renderItem(item, index));
+    return items.map((item: IExtendedPersonaProps, index: number) => this._renderItem(item, index));
   };
 
   // tslint:disable-next-line:no-any
-  private _renderItem(item: any, index: number): JSX.Element {
+  private _renderItem(item: IExtendedPersonaProps, index: number): JSX.Element {
     const { removeButtonAriaLabel } = this.props;
+    const expandGroup = this.props.onExpandGroup;
     const props = {
       item,
       index,
       key: item.key ? item.key : index,
       selected: this.selection.isIndexSelected(index),
-      onRemoveItem: () => this.removeItem(item),
+      // TODO removeItem is incorrectly typed. Remove this cast in the fix.
+      onRemoveItem: () => this.removeItem(item as any),
```

Comment:
This isn't needed if ##8258 merges first

Reply:
** this cast

Reply:
@Adjective-Object looks like we have all checks completed on https://github.com/OfficeDev/office-ui-fabric-react/pull/8260 and would go in first. Would you need to make any changes here, in that case?

Reply:
@Adjective-Object, I've merged #8260. Can you remove the cast and then we'll get this merged as well? Thank you!